unstructured/partition/pdf.py

```diff
@@ -464,21 +464,50 @@ def convert_pdf_to_images(
             yield image
 
 
-def add_pytesseract_bbox_to_elements(elements, bboxes, width, height):
+def _get_element_box(
+    boxes: List[str],
+    char_count: int,
+):
+    """Helper function to get the bounding box of an element.
+
+    Args:
+        boxes (List[str], optional): Defaults to [].
+        char_count (int, optional): Defaults to 0.
+    """
+    min_x = float("inf")
+    min_y = float("inf")
+    max_x = 0
+    max_y = 0
+    for box in boxes:
+        _, _x1, _y1, _x2, _y2, _ = box.split()
+
+        x1, y1, x2, y2 = map(int, [_x1, _y1, _x2, _y2])
+        min_x = min(min_x, x1)
+        min_y = min(min_y, y1)
+        max_x = max(max_x, x2)
+        max_y = max(max_y, y2)
+
+    return ((min_x, min_y), (min_x, max_y), (max_x, max_y), (max_x, min_y)), char_count
+
+
+def add_pytesseract_bbox_to_elements(
```

Comment:
Should it be `add_pytesseract_bboxes_to_elements`? On the other hand the name `bboxes` is confusing to me for a string, although I'm not familiar with the output of `pytesseract.image_to_boxes`.

Reply:
Will change the name. "bbox" stands for "bounding box", which I saw used as the var name somewhere while working on this

Reply:
I understand the abbreviation, but I would expect a variable called `bboxes` to contain a sequence of geometric objects representing bounding boxes... not a string.

Reply:
Ahh.. okay. Pytesseract returns a giant string from `image_to_boxes`. Open to recommendations for another name, though.

Reply:
Ok, how about we just change `bboxes -> bboxes_string`.